utils.ts

```diff
@@ -43,8 +45,13 @@ export class OrderedSet<T> extends Set<T> {
  */
 export function generateHotKeyText(hotkey: Hotkey): string {
     var hotKeyStrings: string[] = [];
-    for (const mod of hotkey.modifiers) {
-        hotKeyStrings.push(MODIFIER_ICONS[mod])
+    if (hotkey.modifiers.length === 4) {
+        // for Mac users using the Hyper Key https://holmberg.io/hyper-key/
+        hotKeyStrings.push(MODIFIER_ICONS["Capslock"]);
```

Comment:
I have three concerns about this. 

1. My biggest is that we cannot tell if the user is actually using a hyper key. They may just actually want to use all of those modifier keys. It's not comfortable on qwerty, but plenty of people use alternative layouts. I don't want to force a UI on people that I can't ensure will benefit from it. That said, I would be open to making this some kind of setting. That way everyone can see the hotkeys as they would like
2. Just checking the length of the modifiers is scary to me. I might be fine, but I would not be surprised if people we using custom mouse buttons as modifier keys.
3. If we could somehow determine a person was wanting to use their hyper key, can we be sure that they have actually mapped it to the caps lock key? It seems like it could be any key.

Reply:
true, I haven't think of the possibility of someone using other methods for multiple modifiers. The way I see it, there should be two possibilities then: 
1) Detect whether a user has karabiner elements installed and the respective setting set, it's readable in `/.config/karabiner/karabiner.json`. However, I have no idea how to make Obsidian read external data in this way.
2) Simply set a toggle in the plugin's settings tab.

Reply:
I think a setting is a good low-effort workaround for now. If there becomes more of a demand for this kind of thing I can build out a more robust settings UI, but for now I think a toggle will be fine. I'll add that to my queue of work

Reply:
thanks :)